freqtrade/exchange/okex.py

```diff
@@ -46,3 +47,37 @@ def _lev_prep(
                     "mgnMode": self.margin_mode.value,
                     "posSide": "long" if side == "buy" else "short",
                 })
+
+    def get_max_pair_stake_amount(
+        self,
+        pair: str,
+        price: float,
+        leverage: float = 1.0
+    ) -> float:
+
+        if self.trading_mode == TradingMode.SPOT:
+            return float('inf')  # Not actually inf, but this probably won't matter for SPOT
+
+        if pair not in self._leverage_tiers:
+            return float('inf')
+
+        pair_tiers = self._leverage_tiers[pair]
+        return pair_tiers[-1]['max'] / leverage
+
+    def load_leverage_tiers(self) -> Dict[str, List[Dict]]:
+        if self.trading_mode == TradingMode.FUTURES:
+            markets = self.markets
+            symbols = []
+
+            for symbol, market in markets.items():
+                if (market["swap"] and market["linear"]):
+                    symbols.append(market["symbol"])
+
+            tiers = {}
+            for symbol in symbols:
+                res = self._api.fetchLeverageTiers(symbol)
+                tiers[symbol] = res[symbol]
```

Comment:
Here for the link

Reply:
at the moment there isn't - but my idea was to run a method (when futures mode) which passes the pairlist in - so we only fetch the pairs necessary. Obviously that method is "empty" for most exchanges (or disabled?) and only fetches the data for exchanges which need it (i guess based on the _ft_has entry ...)